code/discriminative.py

```diff
@@ -0,0 +1,127 @@
+from model import Model
+import torch
+import torch.nn as nn
+import torch.autograd as autograd
+import torch.optim as optim
+import utils
+import numpy as np
+import time
+import constants
+import random
+
+class DiscriminativeModel(nn.Module, Model):
```

Comment:
As a temporary workaround, renaming this to Model4 would allow you to run this via the command line without making changes to ```main.py```
@mudit1990 Do you think this is a good idea?

Reply:
actually, I was thinking the other way round ... let's give our models meaningful names
Model 1,2,3,4 will get very confusing for us ... especially if we are supplying command line args
This can also be run through command line without making any changes in main.py. You just have to pass 'DiscriminativeModel' model as model name

Reply:
I agree about proper names,
but I tried with 'DiscriminativeModel', but it gave key not found error.
Let me try again.

Reply:
have you put the import statement in main.py?
like this one?
from model3 import Model3